iroh-store/src/store.rs

```diff
@@ -366,11 +367,14 @@ impl<'a> WriteStore<'a> {
         let mut total_blob_size = 0;
 
         let mut batch = WriteBatch::default();
+        let mut cid_tracker: HashSet<Cid> = std::collections::HashSet::default();
```

Comment:
I would use an AHashSet ( https://docs.rs/ahash/latest/ahash/ ) since it is faster.

Dig is quite fond of it, and it is already in our dependencies anyway.

Reply:
🔥